### PR TITLE
Porting GODAS to Orion - Part1

### LINKED ISSUE #164: Transfer of the Workflow to Orion
Update the workflow to be able to run on Orion, with the same functionality as at Hera.

@JianKuang-NOAA let's talk on the phone, before starting the porting.

### PR DESCRIPTION
**Pull request by flampouris**
## Description

This PR has necessary updates for the installation of the GODAS to be portable to Orion.
Any of these updates should not change the expected outputs.

Even after this PR, the system does not compile at Orion!

### Issue(s) addressed

- fixes #164 

## Testing
The objective of the testing is that the system still can be installed and run on **Hera**. So test GODAS as follows:

@jkbk2004 to test at Hera, please use the python test script.

@hyunchul386 to test at Hera, please use the README instructions.

### CODE REVIEW

File: test/test.setup_godas.py [hunk 1]
```diff
@@ -1,215 +1,272 @@
...
+    BUILD_COMPILER = input.get('BUILD_COMPILER')
+    MACHINE_ID = input.get('MACHINE_ID')
     USER = os.getenv('USER')
     os.environ["CLONE_DIR"] = CLONE_DIR
+    os.environ['MACHINE_ID'] = MACHINE_ID
     path = os.getcwd()
-    
+
+    print ('USER:'+USER)
+    print ('WORKFLOW_NAME'+WORKFLOW_NAME)
```
Comment (flampouris) [inline test/test.setup_godas.py:119]: Delete Lined 118-119

**Review by jkbk2004 — APPROVED** [review]
System builds and runs ok on hera with the branch!

**Review by hyunchul386 — APPROVED** [review]
By the test it was checked that the current version of GODAS is correctly built and run at Hera .

Just FYI, followings are from ctest of soca_bundle; 
51/789 Test  #51: eckit_test_mpi ..............................................***Timeout 1500.05 sec
87/789 Test  #87: get_eckit_test_data .........................................***Failed    0.58 sec

**Comment by flampouris** [comment]
Thank you both of you. 

Good to know @hyunchul386

**Pull request merged**